external-plugins/rehearse/plugin/handler/handler_test.go

```diff
@@ -510,13 +520,46 @@ Gna meh whatever
 					OrgMembers: map[string][]string{
 						"testorg": {
 							"testauthor",
-							"testuser",
+							userName,
+						},
+					},
+					ChangedFiles:      []string{"changedFile"},
+					TopLevelApprovers: sets.String{},
+					LeafApprovers: map[string]sets.String{
+						"changedFile": {userName: struct{}{}},
+					},
+					ExpectedCanUserRehearse: true,
+				},
+			),
+			Entry("user and author in org - user is a leaf approver - case in OWNERS and GitHub is different for user (1)",
```

Comment:
not sure I understand the `(1)` reference, can you elaborate?

Reply:
Good point - I maybe should be more specific in the naming :smile: